src/style.css

```diff
@@ -1 +1,61 @@
 /* Styles go here */
+.body {
+  font-family: "Roboto", sans-serif;
+  font-size: 13px;
+  font-style: normal;
+  text-transform: uppercase;
+  font-weight: 500;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  cursor: pointer;
+  margin: 20px 30px;
+}
+
+.image {
+  width: 40px;
+  height: 40px;
+  margin: 10px;
+}
+
+.nav {
+  display: flex;
+}
+
+.nav__list {
+  display: flex;
+  margin: 0;
+}
+
+.nav__item {
+  display: flex;
+}
+
+.nav__link {
+  display: inline-block;
+  padding: 22px 10px;
```

Comment:
delete padding left and right, that's why your tests is failing

Reply:
Without this padding, I have this result 
https://prnt.sc/7uKLjWD_qBBa

Reply:
you need to add margin between links